step-templates/roundhouse-database-migration.json

```diff
@@ -22,7 +22,7 @@
r SSL, ignoring setting.\"\n    }\n}\n\n$roundhouseSwitches += $roundhouseServerConnectionString\n\n$roundhouseSwitches += \"--databasetype=$roundhouseDatabaseServerType\"\n$roundhouseSwitches += \"--silent\"\n\n\n# Check for folder definitions\nif (![string]::IsNullOrEmpty($roundhouseUpFolder))\n{\n    # Add up folder\n    $roundhouseSwitches += \"--up=$roundhouseUpFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhouseAlterDatabaseFolder))\n{\n    $roundhouseSwitches += \"--alterdatabasefolder=$roundhouseAlterDatabaseFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhouseRunBeforeUpFolder))\n{\n    $roundhouseSwitches += \"--runbeforeupfolder=$roundhouseRunBeforeUpFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhouseFunctionsFolder))\n{\n    $roundhouseSwitches += \"--functionsfolder=$roundhouseFunctionsFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhouseViewsFolder))\n{\n    $roundhouseSwitches += \"--viewsfolder=$roundhouseViewsFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhouseSprocsFolder))\n{\n    $roundhouseSwitches += \"--sprocsfolder=$roundhouseSprocsFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhouseIndexFolder))\n{\n    $roundhouseSwitches += \"--indexesfolder=$roundhouseIndexFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhouseRunAfterAnyTimeFolder))\n{\n    $roundhouseSwitches += \"--runAfterOtherAnyTimeScriptsFolder=$roundhouseRunAfterAnyTimeFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhousePermissionsFolder))\n{\n    $roundhouseSwitches += \"--permissionsfolder=$roundhousePermissionsFolder\"\n}\n\nif (![string]::IsNullOrEmpty($roundhouseTriggerFolder))\n{\n    $roundhouseSwitches += \"--triggersfolder=$roundhouseTriggerFolder\"\n}\n\nif ([System.Boolean]::Parse($roundhouseDryRun))\n{\n    $roundhouseSwitches += \"--dryrun\"\n}\n\nif ([System.Boolean]::Parse($roundhouseRecordOutput))\n{\n    $roundhouseSwitches += \"--outputpath=$roundhouseOutputPath\"\n}\n\n# Add transaction switch\n$roundhouseSwitches += \"--withtransaction=$($roundhouseWithTransaction.ToLower())\"\n\n# Check for version\nif (![string]::IsNullOrEmpty($roundhouseVersion))\n{\n    # Add version\n    $roundhouseSwitches += \"--version=$roundhouseVersion\"\n}\n\nWrite-Host \"Executing $($roundhouseExecutable.FullName) with $roundhouseSwitches\"\n\n# Execute RoundhousE\nif ($roundhouseExecutable.FullName.EndsWith(\".dll\"))\n{\n\t& dotnet $roundhouseExecutable.FullName $roundhouseSwitches\n}\nelse\n{\n\t& $roundhouseExecutable.FullName $roundhouseSwitches\n}\n\n# If the output path was specified, attach artifacts\nif ([System.Boolean]::Parse($roundhouseRecordOutput))\n{    \n    # Zip up output folder content\n    Add-Type -Assembly 'System.IO.Compression.FileSystem'\n    \n    $zipFile = \"$($OctopusParameters[\"Octopus.Action.Package[RoundhousEPackage].ExtractedPath\"])\\output.zip\"\n    \n\t[System.IO.Compression.ZipFile]::CreateFromDirectory($roundhouseOutputPath, $zipFile)\n    New-OctopusArtifact -Path \"$zipFile\" -Name \"output.zip\"\n}\n"
```

Comment:
I believe on line #20 of your script there's a reference to a _$Version_ variable that I'm unsure of where that's defined?